.buildkite/pipeline-resource-definitions/kibana-console-definitions-sync-8_x.yml

```diff
@@ -0,0 +1,54 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
```

Comment:
We don't need to add another pipeline definition, only to update `branch_configuration: main 8.x` on `.buildkite/pipeline-resource-definitions/kibana-console-definitions-sync.yml` and backport `.buildkite/pipelines/console_definitions_sync.{sh,yml}` + to 8.x.

Apologies for missing that part earlier.  Feel free to ping me if it'll help to iron out the exact details quicker.

Reply:
No worries, I didnt know you could specify two different branches for branch_configuration. This makes everything much easier!